libraries/lib-datahandler/src/main/thrift/sw360.thrift

```diff
@@ -49,10 +49,10 @@ enum VerificationState {
 }
 
 struct VerificationStateInfo {
-    1: required string checkedOn,
-    2: required string checkedBy,
-    3: optional string comment,
-    4: required VerificationState verificationState,
+  1: required string checkedOn,
+  2: required string checkedBy,
+  3: optional string comment,
+  4: required VerificationState verificationState,
```

Comment:
I dunno about nicer, but you didn't add to consistent formatting, obviously. But ok, fine. Let's have it nicer at least in some places